cg/apps/scout/scoutapi.py

```diff
@@ -255,27 +227,15 @@ def upload_fusion_report(
         upload_command.extend([case_id, report_path])
 
         try:
-            LOG.info("Uploading fusion report %s to case %s", report_path, case_id)
+            LOG.info(f"Uploading fusion report {report_path} to case {case_id}")
             self.process.run_command(upload_command)
-        except CalledProcessError:
-            raise ScoutUploadError("Something went wrong when uploading fusion report")
+        except CalledProcessError as error:
+            raise ScoutUploadError("Something went wrong when uploading fusion report") from error
 
     def upload_splice_junctions_bed(self, file_path: str, case_id: str, customer_sample_id: str):
-        """Load a splice junctions bed file into a case in the database.
-
-        Args:
-            file_path           (string):       Path to delivery report
-            case_id             (string):       Case identifier
-            customer_sample_id  (string):       Customers sample identifier
-        Returns:
-            updated_case(dict)
+        """Load a splice junctions bed file into a case in the database."""
 
-        """
-
-        # This command can be run with
-        # `scout update individual -c <case_id> -n <customer_sample_id> splice_junctions_bed
-        #   <path/to/junction_file.bed>`
-        upload_command = [
+        upload_command: List[str] = [
             "update",
```

Comment:
Maybe an update_sample/update_individual function could be created and be reused, similar to upload_report. 

`def update_sample(case_id: str, sample_display_name: str, key: str, value: str)`